sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -170,6 +171,7 @@ case class DropFunctionCommand(
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val catalog = sparkSession.sessionState.catalog
+
```

Comment:
Have you tried `drop function '='`? Does Spark fail with "function not found" or "Cannot drop native function"?

Reply:
> Have you tried `drop function '='`? Does Spark fail with "function not found" or "Cannot drop native function"?

yes.
```
Error in query: Function 'default.=' not found in database 'default';
```

Reply:
OK this can be improved later. Let's leave it for now. Thanks for the investigation!